plans/create_docker_cluster.pp

```diff
@@ -0,0 +1,13 @@
+plan kubernetes::create_cluster(
+  TargetSpec $controller,
+  String[1] $k8s_version='1.2.0',
+  String[1] $k8s_provider='vagrant',
+  String[1] $num_nodes = '3'
+) {
+ 
+  # These get automagically set, right?
+  #run_command($controller, "export $PT_os_family='linux'")
+  #run_command($controller, "export $PT_k8s_version=${k8s_version}")
+  run_command($controller, "export $KUBERNETES_PROVIDER=${kubernetes_provider}")
```

Comment:
I'm not sure whether this is going to work. `export` only applies within your current shell, and you may get a new shell every time you `run_command`.

Reply:
Also, you don't prefix environment variables with `$`.